test_scripts/centos/Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM centos:latest
+FROM centos:8
 
 COPY . /app
-RUN yum -y install initscripts && yum clean all
+RUN yum -y install initscripts procps which && yum clean all
```

Comment:
I think we should do a yum update as well? Otherwise you can end up with old metadata.

Reply:
Good catch